services/community-site/src/pages/RunNode/Screen/UpdateValidator.tsx

```diff
@@ -0,0 +1,226 @@
+  ) => {
+    event.preventDefault();
+    const regex =
+      /^(ftp|http|https):\/\/[\w-]+(\.[\w-]+)+([\w.,@?^=%&amp;:/~+#-]*[\w@?^=%&amp;/~+#-])?$/;
+
+    if (!regex.test(endpoint)) {
+      setEndpointError('Website is required and must be a valid url');
+      return;
+    }
+
+    if (endpoint.length > MAX_ENDPOINT_LENGTH) {
+      setEndpointError(`Website is required and can be ${MAX_ENDPOINT_LENGTH} characters at most`);
+      return;
+    }
+    if (!description || description.length > MAX_DESCRIPTION_LENGTH) {
+      setDescriptionError(
+        `Description is required and can be ${MAX_DESCRIPTION_LENGTH} characters at most`,
+      );
+      return;
+    }
+
+    setDescriptionError('');
+    setEndpointError('');
+    try {
+      let res;
+      if (description && endpoint) {
+        res = await setValidatorInfo(validator.address, description, endpoint);
+        await res.wait();
+        closeEditValidator(true);
+        return;
+      }
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e);
+    }
+  };
+
+  return (
+    <div className="editValidatorScreen">
+      {isUpdatingCommission && (
+        <Modal
+          id="signinModal"
+          title="Update Commission"
+          show={isUpdatingCommission}
+          children={
+            <UpdateCommission
+              id={validator.address}
+              currentCommission={validator.commission}
+              onSuccess={() => {
+                setIsUpdatingCommission(false);
+                closeEditValidator(true);
+              }}
+              isMainnet
+            />
+          }
+          parentElementID="root"
+          onRequestClose={() => {
+            setIsUpdatingCommission(false);
+          }}
+          closeIcon={CloseIcon}
+        />
+      )}
+      <Title title={`Edit validator ${validator.address}`} />
+      <form onSubmit={submit}>
+        <div className="editProfileForm">
+          <div className="formInputContainer">
+            <div>
+              <Text
+                className="profile-inputLabel"
+                label="Validator description (optional)"
+                variant="body2"
+                color="primary"
+              />
+              <InputField
+                error={!!descriptionError}
+                helperText={descriptionError}
+                type="string"
+                className="profileInput"
+                label=""
+                color="secondary"
+                value={description}
+                variant="standard"
+                onChange={(event: any) => {
+                  setDescription(event.target.value);
+                }}
+                margin="normal"
+              />
+            </div>
+          </div>
+          <div className="formInputContainer">
+            <div>
+              <Text
+                className="profile-inputLabel"
+                label="Validator Endpoint (optional)"
```

Comment:
From a contract perspective, they both are required, however, we repopulate them with the current values so that either one or the other could be set. What would be the best wording for that?

Reply:
If they are required, we should remove the "optional"